server_environment_ir_config_parameter/models/ir_config_parameter.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models, _, SUPERUSER_ID
+from openerp.exceptions import UserError
+from openerp.addons.server_environment import serv_config
+
+
+SECTION = 'ir.config_parameter'
+
+
+class IrConfigParameter(models.Model):
+
+    _inherit = 'ir.config_parameter'
+
+    def get_param(self, cr, uid, key, default=False, context=None):
+        value = super(IrConfigParameter, self).get_param(
+            cr, uid, key, default=None, context=context)
```

Comment:
`default=None` is on purpose. The test for default value is at the end of the method.